### PR TITLE
PR for Filter

### PR DESCRIPTION
**Pull request by manismk**
- Added trending, latest filter

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -11,3 +11,4 @@ export { CommentInput } from "./comment-card/CommentInput";
 export { Comment } from "./comment-card/Comment";
 export { ReplyComment } from "./comment-card/ReplyComment";
```
Comment (saketkum11) [inline src/components/index.js:12]: Good use of barrel method

**Review by saketkum11 — APPROVED** [review]
- UI Clean
- Much interactive

**Review by Mandhularajitha — APPROVED** [review]
it looks clean 
clean ui

**Review by aniketxparihar — APPROVED** [review]
- Post Cards can be of same dimensions
- UI is clean
- Code is structured and Modular
- Good CSS naming used
- Good to merge

**Pull request merged**